CRM/Contact/Selector.php

```diff
@@ -1063,7 +1063,8 @@ public function fillupPrevNextCache($sort, $cacheKey, $start = 0, $end = self::C
 
     if (Civi::service('prevnext') instanceof CRM_Core_PrevNextCache_Sql) {
       // SQL-backed prevnext cache uses an extra record for pruning the cache.
-      Civi::cache('prevNextCache')->set($cacheKey, $cacheKey);
+      // Also ensure that caches stay alive for 2 days as per previous code
+      Civi::cache('prevNextCache')->set($cacheKey, $cacheKey, 60 * 60 * 24 * 2);
```

Comment:
I'll give this an `r-run` as is. 

Small `r-code` point: Since the number ("2 days") is used in three different places, I'd recommend extracting to a constant (e.g. `CRM_Core_PrevNextCache_Interface::DEFAULT_TTL`). It could be fiddled into something fancier, but I think a `const` covers us for know and is amenable to future grepping/tweaking (if needed).

Reply:
Good point i have updated to usea cacheDays constant on the SQL interface rather than the general interface as redis already has its own TTL